src/locale/lang/fr.json

```diff
@@ -28,6 +28,7 @@
             "authorize": "Vous ne pouvez associer votre boutique qu'à un seul compte. Choisissez le bien !",
             "authorized": "Votre boutique est associée au compte PrestaShop :  ",
             "connectButton": "Associer",
+            "reonboardTooltip": "Réassocier votre boutique",
```

Comment:
Put the same key in the english file to avoid the small button bug while waiting for the translations ;)